containers/org.eclipse.linuxtools.docker.ui/src/org/eclipse/linuxtools/docker/ui/launch/ContainerLauncher.java

```diff
@@ -2048,20 +1460,8 @@ private void setConsole(RunConsole cons) {
 	 *
 	 * @since 3.0
 	 */
-	public Set<String> getCopiedVolumes(String connectionName,
```

Comment:
Hi Morty.  You cannot remove or modify public APIs.  This would require a major version bump which we don't want to do.  You can add APIs as we are doing a minor release.  So, this API needs to be preserved in the new model.